RecoHGCal/TICL/test/ticl_iterations.py

```diff
@@ -0,0 +1,135 @@
+import FWCore.ParameterSet.Config as cms
+
+from RecoLocalCalo.HGCalRecProducers.HGCalUncalibRecHit_cfi import *
+from RecoLocalCalo.HGCalRecProducers.HGCalRecHit_cfi import *
+
+# patch particle flow clusters for HGC into local reco sequence
+# (for now until global reco is going with some sort of clustering)
+from RecoParticleFlow.PFClusterProducer.particleFlowRecHitHGC_cff import *
+from RecoParticleFlow.PFClusterProducer.particleFlowClusterHGC_cfi import *
+from RecoLocalCalo.HGCalRecProducers.hgcalLayerClusters_cff import hgcalLayerClusters
+from RecoLocalCalo.HGCalRecProducers.hgcalMultiClusters_cfi import hgcalMultiClusters
+
+
+from RecoHGCal.TICL.tracksters_cfi import tracksters
+from RecoHGCal.TICL.FilteredLayerClusters_cfi import FilteredLayerClusters
+from RecoHGCal.TICL.multiClustersFromTracksters_cfi import multiClustersFromTracksters
+
+
+def TICL_iterations_withReco(process):
+  process.FEVTDEBUGHLTEventContent.outputCommands.extend(['keep *_MultiClustersFromTracksters*_*_*'])
+
+  process.FilteredLayerClustersMIP = FilteredLayerClusters.clone(
+      ClusterFilter = "ClusterFilterBySize",
+      algo_number = 8,
+      max_cluster_size = 2, # inclusive
+      iteration_label = "MIP"
+  )
+
+  process.TrackstersMIP = tracksters.clone(
+      original_layerclusters_mask = cms.InputTag("hgcalLayerClusters", "InitialLayerClustersMask"),
+      filtered_layerclusters_mask = cms.InputTag("FilteredLayerClustersMIP", "MIP"),
+      algo_verbosity = 0,
+      missing_layers = 3,
+      min_clusters_per_ntuplet = 15,
```

Comment:
here and in all instances the value 15  is used, while the default is set to 10 in the fillDescriptions.
Please define a more appropriate default in the fillDescriptions and then there is no need to repeat the value in the clones

Reply:
Actually, 10 is a more sound default wrt 15 and recent studies seem to point into this direction. We will update these redefinitions in coming PRs, or eventually set a more appropriate default as soon as we have a better understanding.